browser-test/src/support/index.ts

```diff
@@ -287,17 +287,20 @@ export const closeWarningMessage = async (page: Page) => {
   }
 }
 
+/**
+ * Run accessibility tests using axe accessibility testing engine
+ * @param {Page} page Playwright page to operate against
+ */
 export const validateAccessibility = async (page: Page) => {
-  await test.step('Validate accessiblity', async () => {
-    const results = await new AxeBuilder({page}).analyze()
-    const errorMessage = `Found ${results.violations.length} axe accessibility violations:\n ${JSON.stringify(
-      results.violations,
-      null,
-      2,
-    )}`
-
-    expect(results.violations, errorMessage).toEqual([])
-  })
+  await test.step(
+    'Validate accessiblity',
+    async () => {
+      const results = await new AxeBuilder({page}).analyze()
+      const errorMessage = `Found ${results.violations.length} axe accessibility violations\nOn page: ${page.url()}`
```

Comment:
Removed the JSON.stringify because it was redundant. The same json blob gets printed by the expect method.